pitr/ddl.go

```diff
@@ -51,6 +51,11 @@ type DDLHandle struct {
 	tidbServer *tidblite.TiDBServer
 
 	historyDDLs []*model.Job
+
+	lastDBInfoMap sync.Map
```

Comment:
will this be accessed concurrently?

Reply:
Em... actually, won't.

Reply:
we do not need to define it as sync.Map, it's inefficient